ivy/functional/frontends/numpy/random/functions.py

```diff
@@ -233,3 +233,27 @@ def triangular(left, mode, right, size=None):
         right - (right - mode) * ((1 - u) * (right - mode) / (right - left)) ** 0.5
     )
     return ivy.where(condition, values1, values2)
+
+
+@to_ivy_arrays_and_back
+@from_zero_dim_arrays_to_scalar
+def noncentral_f(dfnum, dfden, nonc, size=None, seed=None):
```

Comment:
[`np.noncentral_f` ](https://numpy.org/doc/stable/reference/random/generated/numpy.random.noncentral_f.html)function doesn't seem to thave the `seed` parameter. Could you please remove this?

Reply:
Hi @hello-fri-end , I have removed this.